chapters/appendix-X-custom-license-namespaces.md

```diff
@@ -0,0 +1,48 @@
+# Appendix X: Custom License Namespaces
+
+## Rationale
+
+The [SPDX License List](https://spdx.org/licenses) defines a curated subset of the most commonly used free software / open source licenses, together with other licenses that have been used in historical, prominent and/or widely used projects for other forms of open content. When a license is added to the SPDX License List, it receives a stable license identifier aligned to a specified license text, with markup and matching guidelines to capture non-substantive variations to the text. The SPDX Legal Team maintains the License List at https://github.com/spdx/license-list-XML and enables community participation in the review and addition of licenses.
+
+The SPDX specification also defines a mechanism for creators of SPDX documents to define custom license texts and identifiers. This enables SPDX documents to refer to licenses that might never be included on the SPDX License List, for a variety of reasons (whether due to limited breadth of use, failure to meet the [license inclusion principles](https://github.com/spdx/license-list-XML/blob/master/DOCS/license-inclusion-principles.md), or otherwise).
+
+These custom license identifiers are prefixed with `LicenseRef-` as defined in [Appendix IV of the SPDX 2.2 Specification](./appendix-IV-SPDX-license-expressions.md). The `LicenseRef-` identifier format was originally assumed to be used in the context of an SPDX document, so that the license text for that identifier would be specified in a corresponding [Other Licensing Information](./6-other-licensing-information-detected.md) section.
```

Comment:
Should only define LicenseRef-'s within own namespace definition document -- thanks Carmen!

Reply:
it's already specified:
> the namespace maintainer creates an SPDX document defining license texts and corresponding identifiers *with the DNS name included*
so example.org cannot publish licenses of somebodyelse.org

Reply:
I've made this a bit more explicit, by adding the following (which is referring to the document created by the namespace maintainer):

> The SPDX document should not define any licenses with identifiers other than those in its own namespace.

I think this should address the concern indicated on the call -- thanks @carmenbianca and @zvr!